src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ReviewCard = ({ review }) => {
+  console.log(review);
```

Comment:
please make sure you aren't pushing console logs to GitHub, its risky for merge conflicts later  when you remove them :)